Debugtest.md

```diff
@@ -0,0 +1,14 @@
+# How to Debug Test Failures:
+
+**Since framework code is used as a dependency, can only be debug via main project.**
+
+1. [Failures.html](target/cucumber-reports/advanced-reports/cumumber-html-reports/js/overview-failures.html) in the main project shows the failures of test.
+
+
+## Logs Capture:
+
+1. All the logs will be in **target/e2e-debug** folder, segmented by feature folders and related scenario files.
+
+## Screenshots Capture:
+
+1. Execution flow and screenshots [here](target/cucumber-reports/advanced-reports/cucumber-html-reports)
```

Comment:
This link is broken.

Reply:
Hi @rmstar link might not work if target folder is missing, should we ask a line "build the project using e2e-tests profile to make the below links work"

Reply:
Just specify the path instead of adding a link because this folder is only created when you run the test.
You can change it to something like: "screenshots can be found in target/cucumber-reports/advanced-reports/cucumber-html-reports"

Reply:
Done.